MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -244,16 +246,55 @@ private static int generateForces(AtBDynamicScenario scenario, AtBContract contr
             effectiveUnitCount = calculateEffectiveUnitCount(scenario, campaign);
 
             for (ScenarioForceTemplate forceTemplate : currentForceTemplates) {
-                generatedLanceCount += generateForce(scenario, contract, campaign,
+                if (forceTemplate.getGenerationMethod() == ForceGenerationMethod.FixedMUL.ordinal()) {
+                    generatedLanceCount += generateFixedForce(scenario, contract, campaign, forceTemplate);
+                } else {
```

Comment:
This should also null check ScenarioForceTemplate::getFixedMul, or the field needs to be validated to ensure it is not null